cloud_governance/policy/ec2_idle.py

```diff
@@ -147,17 +147,18 @@ def __trigger_mail(self, user: str, resource_id: str, metrics: list):
             receivers_list = [f'{self._literal_eval()[user]}@redhat.com']
         else:
             receivers_list = [f'{user}@redhat.com']
-        subject = f'Instance- {resource_id} idle more than {self.INSTANCE_IDLE_MAIL_NOTIFICATION_DAYS} days'
+        subject = f'cloud-governance alert: Stopped ec2-idle more than 4 days'
         body = f"""
 Hi,
 
-Instance: {resource_id} is idle since {self.INSTANCE_IDLE_MAIL_NOTIFICATION_DAYS}.  
-After {self.STOP_INSTANCE_IDLE_DAYS} days of instance is idle we stop that instance. If you don't want to stop this instance add Policy=Not_Delete tag to your instance. 
+Instance: {resource_id} is idle more than {days} days.
+This instance will be stopped. 
+If you you want that cloud-governance will not stop it add Policy=Not_Delete tag to your instance. 
```

Comment:
If you want that cloud-governance will not stop it add Policy=Not_Delete tag to your instance.

Reply:
Both are same